iOS/DuckDuckGo/Subscription/ViewModel/SubscriptionFlowViewModel.swift

```diff
@@ -146,67 +146,43 @@ final class SubscriptionFlowViewModel: ObservableObject {
 
     @MainActor
     private func handleTransactionError(error: UseSubscriptionError) {
-
-        var isStoreError = false
-        var isBackendError = false
```

Comment:
These are gone since store error and backend error pixels are fired in precise use cases. The general idea is that we should only fire a single pixel from calling this function.

Reply:
It's very difficult to review this part, all seems ok but I guess we'll know when we start analysing error pixels...